data_steward/tools/create_tier.py

```diff
@@ -254,16 +256,29 @@ def parse_deid_args(args=None):
                         action='store_true',
                         required=False,
                         help='Log to the console as well as to a file.')
+    parser.add_argument(
+        '-qa',
+        '--qa_handoff_date',
+        dest='qa_handoff_date',
+        help='Will default to today\'s date for QA dataset handoff',
+        default=datetime.today(),
+        required=False)
```

Comment:
You don't need to input the date from the command line.  You can just use the date from a python utility.  This is an interpreter example code.

```
>>> from datetime import datetime
>>> datetime.now()
datetime.datetime(2021, 3, 8, 17, 12, 14, 553973)
>>> # one line to get the current date
>>> datetime.strftime(datetime.now(), '%Y-%m-%d')
'2021-03-08'
```